addons/maintenance/models/maintenance.py

```diff
@@ -190,7 +190,9 @@ def _read_group_category_ids(self, categories, domain):
         """ Read group customization in order to display all the categories in
             the kanban view, even if they are empty.
         """
-        category_ids = categories._search([], order=categories._order, access_rights_uid=SUPERUSER_ID)
+        # search with global ir.rules only
```

Comment:
You are still keeping others (no-global) ir.rule too, or I need a coffee ?

Reply:
This somehow reproduces the existing implementation of `access_rights_uid`.  And that implementation was imperfect: it uses _both_ global and domain-specific rules, indeed.  We don't have an API to get the domain of global rules only.  Do you think we should add one?

Reply:
> Do you think we should add one?

Maybe :thinking: , but not in this PR anyway. Indeed, it is correct to reproduce the existing behavior :+1: but the comment should be correct: `# search with ir.rules but bypass the ACL`